frontend/src/app/pages/Data/Features/NationalMap/index.tsx

```diff
@@ -405,14 +447,16 @@ export const NationalMap = props => {
           </div>
           {loading && <Loader />}
           <Charts
-            // {...state}
             highlight={() => { }}
             select={() => { }}
             selectedFacilities={state.selectedFacilities}
             chartExpanded={chartExpanded}
             setChartExpanded={setChartExpanded}
-          // highlight={hour => _onHighlight(hour)}
-          // select={hour => _onSelect(hour)}
+          />
```

Comment:
I found Charts to be really difficult to understand and it did a lot of complex looping through data that I was hoping we can avoid, so I created a similarly styled StateData